src/VisualStudio/Core/Def/Implementation/LanguageClient/AbstractInProcLanguageClient.cs

```diff
@@ -85,9 +86,9 @@ internal abstract class AbstractInProcLanguageClient : ILanguageClient
         /// <summary>
         /// Can be overridden by subclasses to control what capabilities this language client has.
         /// </summary>
-        protected internal abstract VSServerCapabilities GetCapabilities();
+        protected internal abstract Task<VSServerCapabilities> GetCapabilitiesAsync(CancellationToken cancellationToken);
```

Comment:
does this need to be a task?  I looked through and couldn't find any implementation that was actually async

Reply:
yeah, i'm reverting this.